### PR TITLE
Remove billing organization

### PR DESCRIPTION
**Pull request by nickpell**
Once organizations are officially killed, remove the references to them in billing.

I added a check that no data is being lost (although I highly doubt that any data is in this column on any of the machines) and made the migration reversible.

### CODE REVIEW

**Comment by nickpell** [comment]
https://github.com/dimagi/commcare-hq/pull/8739 merged

**Comment by nickpell** [comment]
@esoergel

**Comment by nickpell** [comment]
@benrudolph @proteusvacuum

File: corehq/apps/accounting/models.py [hunk 1]
```diff
@@ -1352,11 +1341,6 @@ def send_ending_reminder_email(self):
             raise SubscriptionReminderError(
                 "This subscription has no end date."
             )
-        if self.subscriber.organization is not None:
-            raise SubscriptionReminderError(
-                "This reminder email does not yet handle organization "
-                "subscribers."
-            )
```
Comment (proteusvacuum) [inline corehq/apps/accounting/models.py]: lol

**Comment by proteusvacuum** [comment]
is the `Subscriber` model redundant now? It would require a lot more refactoring, but seems like it is now just a list of domains.

**Comment by nickpell** [comment]
@proteusvacuum That's correct - all it represents are domains.

**Comment by proteusvacuum** [comment]
would it open a can of worms to remove it? :skull:

**Comment by nickpell** [comment]
I think it would be nice to remove it ... but that's beyond the scope of this PR at least

**Comment by proteusvacuum** [comment]
Fewer accounting models = :joy_cat:

**Comment by proteusvacuum** [comment]
but yes, I agree, no need to do it now :+1:

**Comment by esoergel** [comment]
:+1:

**Comment by czue** [comment]
:boom:

**Pull request merged**